### PR TITLE
Pharmacist's Harmacy: Adds new labcoat to chemdrobe

### PR DESCRIPTION
**Pull request by Lutowski**
## About The Pull Request
This PR adds:
Pharmacist's labcoat to chemdrobe & loadout, same appearance as RD labcoat but with a chemist recolor.
Sets taur crop flags on some labcoats.

## How This Contributes To The Nova Sector Roleplay Experience
More customization for chemists and other science adjacent characters.

## Proof of Testing
vasv
csaa

## Changelog

:cl:
add: Pharmacist's labcoat added to chemdrobe & loadout.
/:cl:

### CODE REVIEW

File: modular_nova/master_files/code/modules/clothing/suits/labcoat.dm [hunk 1]
```diff
@@ -25,6 +26,13 @@
 	name = "researcher's labcoat"
 	desc = "A Nanotrasen standard labcoat for researchers in the scientific field."
 	icon_state = "labcoat_regular"
+	gets_cropped_on_taurs = FALSE
+
+/obj/item/clothing/suit/toggle/labcoat/nova/pharm
```
Comment (GoldenAlpharex) [inline modular_nova/master_files/code/modules/clothing/suits/labcoat.dm:31]:
```suggestion
/obj/item/clothing/suit/toggle/labcoat/nova/pharmacist
```

File: modular_nova/modules/loadouts/loadout_items/loadout_datum_suit.dm [hunk 2]
```diff
@@ -124,6 +124,10 @@ GLOBAL_LIST_INIT(loadout_exosuits, generate_loadout_items(/datum/loadout_item/su
 	name = "Researcher's Labcoat"
 	item_path = /obj/item/clothing/suit/toggle/labcoat/nova/regular
 
+/datum/loadout_item/suit/labcoat_pharm
+	name = "Pharmacist's Labcoat"
+	item_path = /obj/item/clothing/suit/toggle/labcoat/nova/pharm
```
Comment (GoldenAlpharex) [inline modular_nova/modules/loadouts/loadout_items/loadout_datum_suit.dm:129]:
```suggestion
/datum/loadout_item/suit/labcoat_pharmacist
	name = "Pharmacist's Labcoat"
	item_path = /obj/item/clothing/suit/toggle/labcoat/nova/pharmacist
```

File: code/modules/vending/wardrobes.dm [hunk 3]
```diff
@@ -560,6 +560,7 @@
 		/obj/item/clothing/under/rank/medical/chemist = 2,
 		/obj/item/clothing/under/rank/medical/chemist/skirt = 2,
 		/obj/item/clothing/suit/toggle/labcoat/chemist = 2,
+		/obj/item/clothing/suit/toggle/labcoat/nova/pharm = 2,
```
Comment (GoldenAlpharex) [inline code/modules/vending/wardrobes.dm:563]: Please modularize this, using the `modular_nova\modules\modular_vending\code\wardrobes.dm` file, putting it under `products_nova`, as we don't allow non-modular wardrobe contents additions anymore.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Just some code nitpicks, sprites look good to me!

File: code/modules/vending/wardrobes.dm (hunk 3 above)
Comment (Lutowski) [reply code/modules/vending/wardrobes.dm]: oups, I forgor. Thanks for pointing this out, should be all good!

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me!

**Pull request merged**